cobc/config.def

```diff
@@ -51,7 +51,10 @@ CB_CONFIG_STRING (cb_reserved_words, "reserved-words", _("use of complete/fixed
 CB_CONFIG_INT (cb_tab_width, "tab-width", 1, 12, CB_XRANGE(1,12),
 	_("set number of spaces that are assumed for tabs"))
 
-CB_CONFIG_INT (cb_config_text_column, "text-column", 72, 255, CB_XRANGE(72,255),
+CB_CONFIG_INT (cb_config_indicator_column, "indicator-column", 1, 16, CB_XRANGE(1,16),
```

Comment:
please adjust the text here and for the next to the style "indicator column number for fixed-form reference-format", and for the previous drop "set "

Reply:
... and as all dialects use the same version it seems more reasonable to have that in flags.def; wouldn't it be reasonable to move `cb_indicator_column` from cobc.c only to flags.def and use this everywhere (which would mean one variable less and no changes under "config" at all)?